django-env/Scripts/activate

```diff
@@ -0,0 +1,66 @@
+# This file must be used with "source bin/activate" *from bash*
```

Comment:
We can just delete the django-project folder as this seems to work a lot better. Can do it in this pull request or just do it in main

Reply:
Okay I'll delete it in this PR